src/praisonai-agents/praisonaiagents/config/feature_configs.py

```
@@ -1477,7 +1477,7 @@ class AutonomyLevel(str, Enum):
 HooksParam = Union[List[Any], HooksConfig]
 SkillsParam = Union[List[str], SkillsConfig]
 AutonomyParam = Union[bool, Dict[str, Any], "AutonomyConfig"]
-ToolSearchParam = Union[bool, str, Dict[str, Any], ToolSearchConfig]
+ToolSearchParam = Union[bool, str, Dict[str, Any], "ToolSearchConfig"]
```

Ruff (0.15.21)

[error] Undefined name `ToolSearchConfig`

(F821)